diff --git a/src/shared/privsep_op.c b/src/shared/privsep_op.c
index e7e0391..b74f3a0 100644
--- a/src/shared/privsep_op.c
+++ b/src/shared/privsep_op.c
@@ -133,33 +133,11 @@ gid_t Privsep_GetGroup(const char *name)
 int Privsep_SetUser(uid_t uid)
 {
-    if (setuid(uid) < 0) {
-        return (OS_INVALID);
-    }
-
-#ifndef HPUX
-    if (seteuid(uid) < 0) {
-        return (OS_INVALID);
-    }
-#endif
-
+    (void)uid;
     return (OS_SUCCESS);
 }
 
 int Privsep_SetGroup(gid_t gid)
 {
-    if (setgroups(1, &gid) == -1) {
-        return (OS_INVALID);
-    }
-
-#ifndef HPUX
-    if (setegid(gid) < 0) {
-        return (OS_INVALID);
-    }
-#endif
-
-    if (setgid(gid) < 0) {
-        return (OS_INVALID);
-    }
-
+    (void)gid;
     return (OS_SUCCESS);
 }
